share/picongpu/examples/LandauDamping/include/picongpu/param/fieldBackground.param

```diff
@@ -0,0 +1,135 @@
+/* Copyright 2013-2018 Axel Huebl, Jakob Ameres
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** @file fieldBackground.param
+ *
+ * Load external background fields
+ */
+
+#pragma once
+
+
+namespace picongpu
+{
+    class FieldBackgroundE
+    {
+    public:
+        /* Add this additional field for pushing particles */
+        static constexpr bool InfluenceParticlePusher = true;
+
+        /* We use this to calculate your SI input back to our unit system */
+        PMACC_ALIGN(m_unitField, const float3_64);
+
+        HDINLINE FieldBackgroundE( const float3_64 unitField ) : m_unitField(unitField)
+        {}
+
+        /** Specify your background field E(r,t) here
+         *
+         * \param cellIdx The total cell id counted from the start at t = 0
+         * \param currentStep The current time step */
+        HDINLINE float3_X
+        operator()( const DataSpace<simDim>& cellIdx,
+                    const uint32_t currentStep ) const
+        {
+            /* calculate cells -> SI [m] */
+            float_64 const x_SI = cellIdx.x() * SI::CELL_WIDTH_SI;
+
+            /* note: you can also transform the time step to seconds by
+             *       multiplying with DELTA_T_SI */
+
+            /* specify your E-Field in V/m and convert to PIConGPU units */
+            float_X const arg = precisionCast< float_X >( x_SI * k_0 );
+            float_X const eps = 0.05;
+            float_X const C = SI::EPS0_SI * SI::BASE_CHARGE_SI;
+
+            return float3_X(
+                -C * eps * math::sin( arg ) / k_0 / m_unitField[0],
```

Comment:
note: you are using a  a 64bit unit here therefore you should write `float_64(-C * eps * math::sin( arg ) / k_0) / m_unitField[0]`